js/components/Chart.js

```diff
@@ -30,6 +41,12 @@ define([
       this.format = params.format;
     }
 		
+		export() {
+			console.warn('Export not implemented');
```

Comment:
I'm not sure why this warning is here? It looks like the export is implemented - maybe it was a placeholder?

Reply:
My bad.  I thought at one point that subclasses would need to implement their own export, but later realized it can be a Chart-level function. I'll remove this.